src/lex.js

```diff
@@ -146,7 +149,7 @@ module.exports = function(script) {
                 }
             }
 
-            tokens.push(new types.IntValue(digits));
+            tokens.push(new tokenTypes.Literal('integer', digits));
```

Comment:
hmm bounds checks are done but only later in the parser. I can change that; I honestly don't know if it's better done in the lexer or parser